src/pages/home/Home.tsx

```diff
@@ -1,13 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 
-import { TestStyledComponents } from './style';
+import Title from '../../components/Title';
+import Button from '../../components/Button';
+import Link from '../../components/Link';
+import Input from '../../components/Input';
+
+import { AppRoute } from '../../services/const';
+
+import { HardPopUpSt } from './style';
 
 const Home = () => {
-	return (
-		<>
-			<TestStyledComponents>Home page</TestStyledComponents>
-		</>
-	);
+  const navigate = useNavigate();
+
+  const [logIn, setLogIn] = useState({ login: '', password: '' });
+
+  const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    console.log(logIn);
+    alert(`логин: ${logIn.login}, пароль: ${logIn.password}`);
+    navigate(AppRoute.REGISTRATION);
+  };
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.name === 'login') {
+      setLogIn({ login: e.target.value, password: logIn.password });
+    }
+    if (e.target.name === 'password') {
+      setLogIn({ login: logIn.login, password: e.target.value });
+    }
+  };
```

Comment:
Тут можно немного упростить 
```suggestion
  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
    const { name, value } = e.target;
  
    setLogIn(prevLogin => ({ ...prevLogin, [name]:  value }));
  };
```